public/app/features/explore/ExploreGraphNGPanel.tsx

```diff
@@ -129,14 +129,10 @@ export function ExploreGraphNGPanel({
           legend={{ displayMode: LegendDisplayMode.List, placement: 'bottom', calcs: [] }}
           timeZone={timeZone}
         >
-          <TooltipPlugin mode="single" timeZone={timeZone} />
           <ZoomPlugin onZoom={onUpdateTimeRange} />
-          <ContextMenuPlugin timeZone={timeZone} />
-          {annotations ? (
-            <ExemplarsPlugin exemplars={annotations} timeZone={timeZone} getFieldLinks={getFieldLinks} />
-          ) : (
-            <></>
-          )}
+          <TooltipPlugin data={data} mode="single" timeZone={timeZone} />
```

Comment:
Plugins that require data should accept original data frame via props and use `useGraphNGContext` for data inspection